scipy/stats/tests/test_binned_statistic.py

```diff
@@ -447,3 +448,27 @@ def test_dd_binnumbers_unraveled(self):
         assert_allclose(bcx, bc2[0])
         assert_allclose(bcy, bc2[1])
         assert_allclose(bcz, bc2[2])
+
+    def test_dd_binned_statistic_result(self):
```

Comment:
I'm thinking the name is not as informative in 2 years time. Can you add a comment line in this test, something to the effect that => 
`#NOTE: tests the reuse of bin_edges from previous call`
?